extensions/scheduler/api/src/main/java/io/quarkus/scheduler/Trigger.java

```diff
@@ -31,7 +31,7 @@ public interface Trigger {
     /**
      * The grace period is configurable with {@link Scheduled#overdueGracePeriod()}.
      * <p>
-     * This method returns {@code false} if the last execution has been skipped.
+     * This method returns {@code false} if the last execution should have been fired but was not.
```

Comment:
We should mention that is wa not fired within the fire time + grace period... I'm not sure about the wording.

Reply:
Any chance we could finalize this?

Reply:
@rmanibus ^ ;-)

Reply:
hi @mkouba @gsmet, I reworded it, let me now if it's ok for you !

Reply:
@mkouba I'm reading this again, and what I wanted to express initially in this message is that If a scheduler execution is skipped, it will not be considered as overdue.

Reply:
I think the following is better:
```java
    /**
     * The grace period is configurable with {@link Scheduled#overdueGracePeriod()}.
     * <p>
     * Skipped executions are not considered as overdue.
     *
     * @return {@code false} if the last execution took place between the expected execution time and the end of the grace period, {@code true} otherwise
     * @see Scheduled#overdueGracePeriod()
     */
```